content/ratatui-is-funded.md

```diff
@@ -0,0 +1,237 @@
+
+We looked into the possibility of doing the same with Drips Network and eventually decided that _if Open Collective is able to create a crypto wallet for us_, that will presumably be the easiest way to send funds from Drips Network to Open Collective.
+
+### **Managing Funds**
+
+Shout out to [Benjamin Nickolls (@benjam)](https://github.com/BenJam) from Open Collective, he greatly helped us to figure out the crypto-related details of things and set up a multi-signature [wallet](https://github.com/ratatui-org/ratatui/pull/994) that is managed by us (2 maintainers) and Open Collective. That's how we managed to get everything _dripping_ 💧
+
+All of these were in the works for the past couple of months. We now have the following setup and everything works like a charm!
+
+![Ratatui Funding Diagram](/ratatui-funding-diagram.png)
+
+You can see the related [transactions](https://opencollective.com/ratatui/transactions) on Open Collective:
+
+![Ratatui Transactions](/ratatui-transactions.png)
+
+---
+
+## **The Future**
+
+Now that we are an open source project and the money is involved, we need to draw some lines about how we manage these funds. Additionally, we need to come up with a thinking model that will be taken as a baseline when it comes to utilizing this financial source.
+
+### **Expenses**
+
+In the simplest terms, the primary and the most traditional reason that companies make money is to stay afloat and expand their _business_.
+
+In our case, these are already semi-covered thanks to open source. We don't need to worry about the project dying, because if that happens tomorrow someone else can fork the project and start Ratatui v2 (which is exactly what we have done).
+
+However, there might be still financial costs of maintaining an open source project. These include things such as a custom domain, additional services/servers, promotional elements (e.g. merch) and so on.
+
+Thus we will be using these funds to cover these maintenance/promotional costs.
+
+### **Sustainability**
+
+As companies receive investment, they just don't go "hey we got the money let's go home y'all".
+
+That is the same in our case as well. We still have a lot to do.
+
+<q>So can't we just pay people to do work?</q>
+
+That's the beauty of open source, people are _already_ doing work. Not because they want to get rich off of the project, but because they want to learn new things and have some fun on the side.
+
+Paying people for their open source work will also push the incentive that everyone might get paid through this work. Which means when there is no money left, there is also no project left to work on.
```

Comment:
Should we mention that there are alternative ways to fund work, e.g. neovim funding one-off organised contributions? Or at least make a disclaimer that this policy works for us and is not a one-size-fits-all solution?

Reply:
d596cae76ea6449e90573f4ad5e5ac732d87cad7